### PR TITLE
Introducing Stockpot

### PR DESCRIPTION
**Pull request by jaysonesmith**
Setting up the foundational work to get Stockpot created and to claim the gem name.

### CODE REVIEW

**Comment by jaysonesmith** [comment]
I've pushed stockpot [v0.16.3.pre](https://rubygems.org/gems/stockpot)! 🎉 @arettberg I've also made changes matching your requests. Let me know if they fit the bill. If I've missed anything, I'll clean it up, just let me know.

File: stockpot/test/dummy/log/development.log [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Started GET "/" for ::1 at 2019-09-23 10:28:14 -0700
```
Comment (arettberg) [inline stockpot/test/dummy/log/development.log]: probably need to tweak a gitignore somewhere so this doesn't get committed

File: stockpot/test/stockpot_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class Stockpot::Test < ActiveSupport::TestCase
+  test "truth" do
+    assert_kind_of Module, Stockpot
+  end
+end
```
Comment (arettberg) [inline stockpot/test/stockpot_test.rb]: let's swap this for rspec

**Comment by jaysonesmith** [comment]
Still currently unable to get rspec working, but have verified I'm able to curl the basic endpoints I've included as expected.

**Comment by jaysonesmith** [comment]
Moving this to its own repo.

**Pull request closed without merging**